composable-ui/package.json

```diff
@@ -27,6 +27,9 @@
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
     "@hookform/resolvers": "^2.9.7",
+    "@klevu/core": "^5.0.0",
+    "@klevu/ui": "^2.0.0",
+    "@klevu/ui-react": "^2.0.0",
```

Comment:
@nikhil-narayana-klevu please remove these dependencies, as they should not be a part of the baseline composable-ui package.json. The developer will install these when following the Klevu install documentation.

Reply:
I have removed these packages. Please check now.